css/tufte.scss

```diff
@@ -179,14 +179,14 @@ p { line-height: 2rem;
     padding-right: 0;
     vertical-align: baseline; }
 
-// blockquote p {  font-size: 1.1rem;
-//                 line-height: 1.78181818;
-//                 margin-top: 1.78181818rem;
-//                 margin-bottom: 1.78181818rem;
-//                 width: 45%;
-//                 padding-left: 2.5%;
-//                 padding-right: 2.5%; }
-//
+blockquote p {  font-size: 1.1rem;
```

Comment:
Blockquote isn't used anywhere else and fits the format I desired, so I uncommented it from the CSS.